recipes/flake8-rst/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flake8-rst" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec9c7b7a5475c0b3ce8aee4db951f33beffad68bdb79d99d45788d4aa083ce52
+
+build:
+  number: 0
```

Comment:
I think this should probably be `noarch: python`:

https://conda-forge.org/docs/meta.html#building-noarch-packages